superflore/generators/bitbake/run.py

```diff
@@ -209,9 +209,13 @@ def main():
 
         if num_changes == 0:
             info('ROS distro is up to date.')
-            info('Exiting...')
-            clean_up()
-            sys.exit(0)
+            summary = overlay.get_change_summary()
+            if len(summary) == 0:
+                info('Exiting...')
+                clean_up()
+                sys.exit(0)
+            else:
+                info('But there are some changes in other regenerated files')
```

Comment:
I would really prefer that this refer to the type of files that did change, and why they changed. Doesn't need to be super detailed or anything, just something along the lines of "Commiting changes to [what] because [why]".